pkg/kor/storageclasses.go

```diff
@@ -14,6 +14,13 @@ import (
 	"github.com/yonahd/kor/pkg/filters"
 )
 
+var exceptionStorageClasses = []ExceptionResource{
```

Comment:
Is this a false positive? Or do you not have any use of the storage class

Reply:
I believe this storage class should be treated as false positive as it comes with the vanilla configuration of minikube.